content/docs/thinking-in-react.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: thinking-in-react
-title: Thinking in React
+title: React में सोचना 
 permalink: docs/thinking-in-react.html
 redirect_from:
   - 'blog/2013/11/05/thinking-in-react.html'
   - 'docs/thinking-in-react-zh-CN.html'
 prev: composition-vs-inheritance.html
 ---
 
-React is, in our opinion, the premier way to build big, fast Web apps with JavaScript. It has scaled very well for us at Facebook and Instagram.
+हमारी राय में, जावास्क्रिप्ट द्वारा, बड़ी और तेज़ वेब एप्पस बनाने का React एक मुख्य तरीका है। यह हमारे लिए Facebook और Instagram के लिए काफी अच्छे से बढ़ा है।
```

Comment:
हमारी राय में, जावास्क्रिप्ट द्वारा, बड़ी और तेज़ वेब एप्पस बनाने का React एक मुख्य तरीका है। changes the meaning of the sentence, reword the order of words

Reply:
React, हमारी राय में, जावास्क्रिप्ट द्वारा, बड़ी और तेज़ वेब एप्पस बनाने का एक मुख्य तरीका है। 
Better?

Reply:
sounds better